### DISCUSSION TITLE
Welcome to voyager Discussions!

### DISCUSSION THREAD

**Discussion by martypitt** [vyneco/voyager#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Vishal1297** [comment]
Hello @martypitt,

I am Vishal, a software developer from India. Currently i am new to this open source community but i want learn and build something
useful with help of this community.

I am curious to know what are your thoughts in building this tool which can create architectural diagram for microservices and
how can i contribute in this?

**Reply by martypitt** [reply]
Hey @Vishal1297 

Thanks for the message

We're working on open sourcing voyager.  Unfortunately, the code is a bit entwined with our other product - [Vyne](https://vyne.co).

Until we get this code open-sourced, contributing on the code isn't possible.  However, there's other ways you can contribute to making Voyager successful:

 * Playing with the tool, and contributing ideas for our roadmap
 * Reporting any bugs
 * Tweeting or Blogging about the tool, so that others become aware.

Hope this is helpful
